A62-pick-first.md

```diff
@@ -9,63 +9,68 @@ A62: `pick_first` LB policy
 
 ## Abstract
 
-This design specifies a configuration option for the `pick_first` LB policy that
-would enable it to randomly shuffle the order of the addresses it receives
-before it starts to attempt to connect.
-
-Given that `pick_first` is the default LB policy in all gRPC implementations, it
-is necessary to have a written specification that details its behavior. This
-document provides that specification.
+This document describes a couple of changes being made to the `pick_first` LB
+policy with regards to
+- Expected behavior when connections to all addresses fail, and
+- Support for address order randomization
 
 ## Background
 
 All gRPC implementations contain a simple load balancing policy named
 `pick_first` that can be summarized as follows:
 - It takes a list of addresses from the name resolver and attempts to connect to
   those addresses one at a time, in order, until it finds one that is reachable.
-  All RPCs sent on the gRPC channel will be sent to this address.
-- If this connection breaks at a later point in time, pick_first will not
-  attempt to reconnect until the application requests that it does so.
+  - All RPCs sent on the gRPC channel will be sent to this address.
```

Comment:
Suggest structuring this list as follows:

- It takes a list of addresses from the name resolver and attempts to connect to
  those addresses one at a time, in order, until it finds one that is reachable.
- Once it finds a reachable address:
  - All RPCs sent on the gRPC channel will be sent to this address.
  - If this connection breaks at a later point in time, `pick_first` will not
    attempt to reconnect until the application requests that it does so, or makes
    an RPC.
- If none of the addresses are reachable, it applies an exponential backoff
  before attempting to reconnect.